rasa_core/server.py

```diff
@@ -146,25 +146,26 @@ def _create_agent(
         return None
 
 
-def create_app(model_directory,
-               interpreter=None,
-               loglevel="INFO",
-               logfile="rasa_core.log",
-               cors_origins=None,
-               action_factory=None,
-               auth_token=None,
-               tracker_store=None
+def create_app(model_directory,           # type: Text
+               interpreter=None,          # type: Optional
+                                        # [Union(text,NL interpreter)]
+               loglevel="INFO",           # type: Optional [text]
+               logfile="rasa_core.log",   # type: Optional [text]
+               cors_origins=None,         # type: Optional [list]
+               action_factory=None,       # type: Optional [text]
+               auth_token=None,           # type: Optional [text]
+               tracker_store=None         # type: Optional[TrackerStore]
                ):
     """Class representing a Rasa Core HTTP server."""
 
     app = Flask(__name__)
     CORS(app, resources={r"/*": {"origins": "*"}})
-
+    # Setting up logfile
     utils.configure_file_logging(loglevel, logfile)
 
     if not cors_origins:
         cors_origins = []
-
+    type(cors_origins)
```

Comment:
I guess this is not needed

Reply:
Okay sure! I'm making the necessary changes as you said